pkgs/applications/science/math/numworks-epsilon/default.nix

```diff
@@ -31,17 +32,18 @@ stdenv.mkDerivation rec {
     python3
     imagemagick
     gcc-arm-embedded
+    python311Packages.lz4
   ];
 
   makeFlags = [
     "PLATFORM=simulator"
   ];
 
-  patches = [
-    # Remove make rule Introduced in cba596dde7
-    # which causes it to not build with nix
-    ./0001-ion-linux-makerules.patch
-  ];
+  # patches = [
+  #   # Remove make rule Introduced in cba596dde7
+  #   # which causes it to not build with nix
+  #   ./0001-ion-linux-makerules.patch
+  # ];
```

Comment:
Why is this commented out? If it doesn't matter you can just remove it. The upstream file is already patched.

Reply:
You're right, forgot to remove it